app/models/adapters/active_admin_pundit_adapter.rb

```diff
@@ -0,0 +1,46 @@
+module Adapters
+  class ActiveAdminPunditAdapter < ActiveAdmin::PunditAdapter
+    def get_policy(subject, user, resource)
+      "ActiveAdmin::#{subject}Policy".constantize.new(user, resource)
+    end
+
+    def retrieve_policy(subject)
```

Comment:
I think `retrieve_policy` and `get_policy` can be private, right?

Reply:
`retrieve_policy` is a [public method in `ActiveAdmin::PunditAdapter`](https://github.com/activeadmin/activeadmin/blob/93175ed7ff09d113cb77f5e946ee8f054c438775/lib/active_admin/pundit_adapter.rb#L32), I'm unsure if making it private will deny access to this function from outside the class when ActiveAdmin tries to call it. `get_policy` definetly can be a private method